package/firmware/ipq-wifi/Makefile

```diff
@@ -34,8 +34,9 @@ ALLWIFIBOARDS:= \
 	dynalink_dl-wrx36 \
 	edgecore_eap102 \
 	edimax_cax1800 \
-        linksys_mx4200 \
+  linksys_mx4200 \
```

Comment:
This seems an oops.

Reply:
This was actually auto formatting I think, will fix

Reply:
fixed, though I see the issue - this line looks super weird in VS Code, has different leading whitespace to all others around it - renders fine on Github but looks way off in the IDE

Reply:
Please do not add unrelated changes.